### PR TITLE
[New Rule] Privilege Escalation via Named Pipe Impersonation

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/604

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
image

File: rules/windows/privilege_escalation_named_pipe_impersonation.toml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/11/23"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies privileges escalation attempt via Named Pipe Impersonation. This is a known technique implemented in several
```
Comment (peasead) [inline rules/windows/privilege_escalation_named_pipe_impersonation.toml:10]:
```suggestion
Identifies a privilege escalation attempt via Named Pipe Impersonation. This is a known technique implemented in several
```

File: rules/windows/privilege_escalation_named_pipe_impersonation.toml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+pentesting frameworks such as Meterpreter getsystem command.
```
Comment (peasead) [inline rules/windows/privilege_escalation_named_pipe_impersonation.toml:11]:
```suggestion
pentesting frameworks such as the Meterpreter getsystem command.
```

**Review by peasead — APPROVED** [review]

File: rules/windows/privilege_escalation_named_pipe_impersonation.toml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2020/11/23"
+maturity = "production"
+updated_date = "2020/11/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a privilege escalation attempt via Named Pipe Impersonation. This is a known technique implemented in several
+pentesting frameworks such as the Meterpreter getsystem command.
```
Comment (bm11100) [inline rules/windows/privilege_escalation_named_pipe_impersonation.toml:10]:
```suggestion
Identifies a privilege escalation attempt via named pipe impersonation. An adversary may abuse this technique by utilizing a framework such Metasploit's meterpreter getsystem command.
```

Comment (bm11100) [reply rules/windows/privilege_escalation_named_pipe_impersonation.toml:10]: re-lint

File: rules/windows/privilege_escalation_named_pipe_impersonation.toml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Privilege Escalation via Named Pipe Impersonation"
+references = ["https://www.ired.team/offensive-security/privilege-escalation/windows-namedpipes-privilege-escalation"]
+risk_score = 71
```
Comment (bm11100) [inline rules/windows/privilege_escalation_named_pipe_impersonation.toml:17]:
```suggestion
risk_score = 73
```

**Review by bm11100 — APPROVED** [review]

**Pull request merged**